azure-pipelines.yml

```diff
@@ -1,38 +1,41 @@
 trigger:
-- master
+- updates/strategy-tweak
 
-jobs:
-- job: Build_Linux_based_images
-  displayName: Build Linux based images
-  pool:
-    vmImage: 'ubuntu-16.04'
-  steps:
-  - template: yml-steps/build-and-push.yml
-    parameters:
-      targetStage: 'linux-arm'
-      repository: 'myapp'
-      dockerfile: 'Dockerfile.Linux'
-  - template: yml-steps/build-and-push.yml
-    parameters:
-      targetStage: 'linux-x64'
-      repository: 'myapp'
-      dockerfile: 'Dockerfile.Linux'
+#need to add in these variables
+variables:
```

Comment:
This looks as if people should be adding these variables through code, which they shouldn't. This should better be a hint at the doc in the readme.md.